makefiles/variables.mk

```diff
@@ -43,8 +43,12 @@ else ifeq ($(YEAR), 2019)
 	MPP_FUNCTION_BACKEND?=enchainement_primitif
 	MPP_FUNCTION?=enchainement_primitif_interpreteur
 else
-    $(warning WARNING: there is no default configuration for year: $(YEAR))
+    $(warning WARNING: there is no default configuration for year: $(YEAR), you have to provide your own source files)
     $(warning WARNING: example specification files and fuzzer tests are not included for year: $(YEAR))
+	SOURCE_FILES=$(call source_dir,$(ROOT_DIR)/ir-calcul/M_SVN/$(YEAR)/)
+	SOURCE_EXT_FILES?=$(call source_dir_ext,$(ROOT_DIR)/m_ext/2022/)
+	MPP_FUNCTION_BACKEND?=enchainement_primitif
+	MPP_FUNCTION?=enchainement_primitif_interpreteur
```

Comment:
So this is a default source for any year not in `2019..2022`, but shouldn't this PR add an official block for year 2023?

Reply:
Yes definitely, I'll add it (the PR was draft and mainly to get our tests running in the first place ;) )